src/main/java/org/checkerframework/specimin/JavaParserUtil.java

```diff
@@ -373,4 +360,101 @@ static String removeMethodReturnTypeAndAnnotationsImpl(String declAsString) {
     String result = methodWithoutReturnType.replace("< ", "<");
     return result;
   }
+
+  /**
+   * Checks to see if an expression (usually a field or method) is static.
```

Comment:
> usually a field or method

Since these are expressions, this must be a _reference_ to a field or method, rather than a field or method definition.